lib/transport_web/templates/user/organization_datasets.html.eex

```diff
@@ -3,7 +3,13 @@
     <p class="message message--info"><%= get_flash(@conn, :info) %></p>
   <% end %>
   <h1><%= dgettext("user", "Second step") %></h1>
-  <h2><%= dgettext("user", "Choose a dataset you want to be on transport.data.gouv.fr") %></h1>
+  <h2><%= dgettext("user", "Choose a dataset you want to be on transport.data.gouv.fr") %></h2>
```

Comment:
I think wording could be clearer if we say something like:

> Choose a dataset to publish

Reply:
We need to re-think wording on every pages